cmd/controller-manager/app/controller-manager.go

```diff
@@ -249,6 +254,16 @@ func setInt(target *int, defaultValue int) {
 
 func setDefaultFederationConfig(fedConfig *corev1a1.FederationConfig) {
 	spec := &fedConfig.Spec
+
+	if len(spec.Scope) == 0 {
+		defaultValue := os.Getenv(defaultScopeEnv)
+		if len(defaultValue) != 0 {
+			if defaultValue != string(apiextv1b1.ClusterScoped) && defaultValue != string(apiextv1b1.NamespaceScoped) {
+				glog.Fatalf("DefaultScope must be Cluster or Namespaced; got %q", defaultValue)
+				spec.Scope = apiextv1b1.ResourceScope(defaultValue)
```

Comment:
This statement needs to appear outside of the conditional block that checks for an invalid scope.  As written, the default scope would never be set.

Reply:
yeah, makes sense. I will place that condition outside of that `if` condition where I'm checking for an invalid scope.